src/commands/impl/moderation/ClearCommand.ts

```diff
@@ -0,0 +1,65 @@
+import Discord from 'discord.js';
+import BotCommand from '../../BotCommand';
+import { config } from '../../../config/amelia.config';
+import MessageService from '../../../services/MessageService';
+import MessageServiceImpl from '../../../services/MessageServiceImpl';
+
+const MAX_MESSAGES = 99;
+
+export default class ClearCommand implements BotCommand {
+  name: string;
+  alias: string[];
+  description: string;
+  usage: string;
+  enabled: boolean;
+  messageService: MessageService;
+
+  constructor() {
+    this.name = 'clear';
+    this.alias = [];
+    this.description = 'Verwijdert een aantal berichten.';
+    this.usage = `${process.env.PREFIX}clear <aantal berichten>`;
+    this.enabled = true;
+    this.messageService = new MessageServiceImpl();
+  }
+
+  execute(msg: Discord.Message, args: string[]): boolean {
+    if (!this.enabled) return false;
+
+    if (args.length === 0) {
+      msg.reply('je moet een aantal opgeven.');
+      return false;
+    }
+
+    const messagesToDelete = parseInt(args[0]);
+    if (messagesToDelete) {
+      if (messagesToDelete > MAX_MESSAGES || messagesToDelete < 0) {
```

Comment:
Puur om te laten zien dat ik er inderdaad naar heb gekeken, time to fuck some ants :magisterguy:

Is het misschien mooier om er `if (messagesToDelete || messagesToDelete < 0)` van te maken ivm je reply?
Als ik -10 invul krijg ik als reply 'je mag maximaal 99 messages clearen', terwijl -10 duidelijk minder is dan 99.

Reply:
deze is fixed